### PR TITLE
UI farm improvements

### PR DESCRIPTION
**Pull request by seeden**
- added fingerprint when the user is trying to create a plot
- used a local version of the local attempted proof table
- used different variant for "Add a Plot" button
- improved Slovak translation
- updated deps

### CODE REVIEW

File: electron-react/src/components/farm/FarmLastAttemptedProof.tsx [hunk 1]
```diff
@@ -35,7 +48,23 @@ export default function FarmLastAttemptedProof() {
...
+        rows={reducedLastAttemtedProof}
+        caption={isEmpty && (
+          <Typography>
+            <Trans id="FarmLastAttemptedProof.emptyDescription">
+              None of your plots have passed the plot filter yet.
+            </Trans>
+            
+            {!!size && (
+              <Trans id="FarmLastAttemptedProof.emptySubDescription">
+                But you are currently farming <FormatBytes value={size} precision={3} />
```
Comment (hoffmang9) [inline electron-react/src/components/farm/FarmLastAttemptedProof.tsx:62]: Need a space here in front of "But"

File: src/wallet/chialisp.py [hunk 2]
```diff
@@ -69,8 +69,21 @@ def sha256(*argv):
...
+
+((c (q ((c 2 (c 2 (c 3 (q ()))))))
+    (c (q ((c (i (l 5)
+                 (q (sha256 (q 2)
+                            ((c 2 (c 2 (c 9 (q ())))))
+                            ((c 2 (c 2 (c 13 (q ())))))))
+                 (q (sha256 (q 1) 5))) 1))) %s)))
+"""
+
+
```
Comment (hoffmang9) [inline src/wallet/chialisp.py:84]: I think this may be a merge error as Richard removed this recently in dev IIRC

**Review by hoffmang9 — CHANGES_REQUESTED** [review]
With these two issues, it otherwise looks good

File: src/wallet/chialisp.py (hunk 2 above)
Comment (seeden) [reply src/wallet/chialisp.py]: I did one more rebase and I hope this is fixed as well

Comment (seeden) [reply src/wallet/chialisp.py]: Anyway, I saw the same version as we have in the dev branch.

**Pull request merged**